packages/react/src/Grid/Grid.tsx

```diff
@@ -48,7 +48,7 @@ export type GridProps = {
    */
   compact?: boolean
   /** The amount of vertical whitespace between rows of the grid. */
-  gapVertical?: 'small' | 'large'
+  gapVertical?: 'small' | 'medium' | 'large'
```

Comment:
So I think `medium` should be the default, and this prop should accept a `none` value.